docs/gettinginvolved/cluster-setup.rst

```diff
@@ -199,6 +171,79 @@ Or you can run, for example, the acceptance tests against the created cluster:
      --flocker-version='' \
      flocker.acceptance.obsolete.test_containers.ContainerAPITests.test_create_container_with_ports
 
+==============================
+Adding containers and datasets
+==============================
+To make it easier to reuse a test cluster, and test under different configurations, Flocker provides a tool to create a certain number of datasets and containers per node.
```

Comment:
Ops, no coma before and. @jml question about re-use. In word reference appears as reuse, and Miriam Webster doesn't find "re-use" and suggests "reuse" instead.
http://www.merriam-webster.com/dictionary/suggestions/re-use
Where is the place to look at that stuff?